loops.js

```diff
@@ -51,3 +66,18 @@
 // >> 9
 // ==========================================
 
```

Comment:
De uitkomst van deze methode is niet helemaal zoals opdracht, bij jou begint de > teken bij getal 4. Als je de teller getallen wijzigt dan klopt ie weer :)